tests/test_symbolics.py

```diff
@@ -643,7 +644,8 @@ def test_pow_precision(dtype, expected):
     op.apply()
 
     assert expected in str(op)
-    assert np.all(f.data == 8.0)
+    tolerance = 1e-8 if dtype is np.float32 else 1e-16
```

Comment:
tolerance = 1e-8 if dtype is np.float32 else 1e-16
```suggestion
    tolerance = 1e-7 if dtype is np.float32 else 1e-16
```